.github/workflows/ci.yml

```diff
@@ -0,0 +1,257 @@
+---
+name: Continuous Integration tests
+
+on: [pull_request]
+
+jobs:
+  main_tests:
+    name: Main tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.9']
+        # ['2.7', '3.6', '3.7', '3.8', '3.9', '3.10.0-rc.1', 'pypy-2.7', 'pypy-3.6', 'pypy-3.7-nightly']
+        include:
+          - os: ubuntu-latest
+            python-version: '3.6'
+          - os: ubuntu-latest
+            python-version: '3.7'
+          - os: ubuntu-latest
+            python-version: '3.8'
+          ## Expected failure
+          # - os: ubuntu-latest
+          #   python-version: '3.10.0-rc.1'
```

Comment:
See: https://github.com/dralley/hpy/pull/1/checks?check_run_id=3286253606

Reply:
this is a failure that we would also have on master, is it correct? I.e. it's an hpy problem, it's not related to the choice of azure vs github actions?

Reply:
I believe so, yes.  Azure Pipelines isn't testing 3.10 RC.

I think I messed up this link though, it doesn't include a 3.10 runner, and I definitely meant to link to one.